pages/news-post.html

```diff
@@ -0,0 +1,211 @@
+            alt="Language Selection"
+            class="cursor-pointer object-contain h-5 w-5"
+          />
+          <div class="absolute left-0 w-full bg-white" id="language-options">
+            <a href="#" class="block">
+              <img
+                src="../images/SVGs/norway.svg"
+                aria-label="Norwegian Flag Language Selection"
+                alt="Norwegian Flag"
+                class="inline h-6 w-8"
+              />
+            </a>
+            <a href="#" class="block">
+              <img
+                src="../images/SVGs/unitedKingdom.svg"
+                aria-label="English Flag Language Selection"
+                alt="English Flag"
+                class="inline h-6 w-8"
+              />
+            </a>
+            <a href="#" class="block">
+              <img
+                src="../images/SVGs/madagascar.svg"
+                aria-label="Malagasy Flag Language Selection"
+                alt="Malagasy Flag"
+                class="inline h-6 w-8"
+              />
+            </a>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- Search Bar Overlay, hidden initially. -->
+    <section
+      id="search-drawer-from-header"
+      class="fixed border border-black inset-x-0 top-0 w-full h-16 bg-white hidden transform -translate-y-full transition-transform duration-500 ease-out z-50"
+      aria-label="search-bar"
+    >
+      <h2 id="search-label" class="sr-only">Search Bar</h2>
+      <div class="flex justify-center items-center pt-3">
+        <div
+          class="w-full max-w-lg mx-auto border border-black bg-gray-100 flex items-center"
+        >
+          <form class="flex flex-grow">
+            <label for="Search-Drawer" class="sr-only">Search Site</label>
+            <input
+              type="search"
+              class="search-drawer_input flex-grow bg-transparent focus:outline-none p-2 text-sm"
+              id="Search-Drawer"
+              placeholder="Search..."
+              autocomplete="off"
+            />
+            <button
+              type="submit"
+              class="search-drawer_submit p-2 text-gray-500 flex items-center justify-center"
+            >
+              <img
+                src="../images/SVGs/Search.svg"
+                class="w-4 h-4"
+                aria-label="Search Now Button"
+                alt="Search Now Button"
+              />
+            </button>
+          </form>
+        </div>
+        <!-- Close button ready for JS. Should also close on outside-click, not just by close button itself -->
+        <button
+          id="closeSearchBarDrawer"
+          class="search-drawer_close text-sm pr-5"
+        >
+          Close
+        </button>
+        <!-- JS intention: On click, add -translate-y-full to section id="search-drawer-from-header" and remove translate-y-0 -->
+      </div>
+    </section>
+    <!-- Section 3 -->
+    <!-- Support us -->
+<section class="flex flex-col gap-20">
```

Comment:
If Support us and Newsletter will be in separate sections, the Support us sections need a h1-h6 or to be changed to a <div> or another element that does not require a h-element inside it.